doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -138,6 +138,12 @@ Then run these commands
 
 #### Configure snmpd
 
+Create a backup of your old snmpd.conf that shipped with your CentOS install.
```

Comment:
Not sure this is needed, if users want the default config back, they can just reinstall snmp.